docs/examples/classification.ipynb

```diff
@@ -0,0 +1,946 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df_train, df_test = train_test_split(df, train_size=0.75, random_state=1234321)\n",
+    "df = (\n",
+    "    pd.concat((df_train, df_test), axis=0, keys=(\"train\", \"test\"))\n",
+    "    .reset_index(level=0)\n",
+    "    .rename(columns={\"level_0\": \"split\"})\n",
+    ")\n",
+    "\n",
+    "y_train, y_test = df_train[y_var], df_test[y_var]\n",
+    "X_train, X_test = df_train[x_vars], df_test[x_vars]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We check whether the split results in two approximately identically distributed samples."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th>binary_response</th>\n",
+       "      <th>False</th>\n",
+       "      <th>True</th>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>split</th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>test</th>\n",
+       "      <td>0.673457</td>\n",
+       "      <td>0.326543</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>train</th>\n",
+       "      <td>0.669891</td>\n",
+       "      <td>0.330109</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "binary_response     False     True \n",
+       "split                              \n",
+       "test             0.673457  0.326543\n",
+       "train            0.669891  0.330109"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "count_by_split = df.groupby(\"split\").apply(len)\n",
+    "grouper = df.groupby([\"split\", y_var])\n",
+    "\n",
+    "(df.groupby([\"split\", y_var]).apply(lambda x: x['count'].sum())\n",
+    " / count_by_split).unstack(y_var)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2. Models\n",
+    "We aim at predicting the conditional expectation $E(Y|X)$ and will finally evaluate the models with the log loss $$S(y, z) = -ylog(z) - (1 - y) log (1 - z)$$\n",
```

Comment:
```suggestion
    "We aim at predicting the conditional expectation $E(Y|X)$ and will finally evaluate the models with the log loss $$S(y, z) = -y \log(z) - (1 - y) \log (1 - z)$$\n",
```
Again, mention "probability".

Reply:
I included the \log and the probability mention in 8d3e7206d7ee6c0c4ab1fa004bdf4dbf33e7a346